tests/transformation_tests/pathfinders/resources/extracted_validated_data.py

```diff
@@ -0,0 +1,166 @@
+import pandas as pd
+
+financial_completion_date = pd.DataFrame({"Pathfinder financial completion date": [pd.Timestamp("2001-01-01")]})
```

Comment:
These are "globally" or at least module level variables and as such should be named in CAPS_CASE